platform-sdk/swirlds-unit-tests/common/swirlds-common-test/src/test/java/com/swirlds/common/test/metrics/extensions/FractionalTimerTests.java

```diff
@@ -41,71 +43,71 @@ void reset() {
+        assertEquals(0.0, metric.getActiveFraction(), "no work has started yet, so the busy fraction should be 0");
+        metric.activate();
         assertEquals(
                 0.0,
-                metric.getBusyFraction(),
+                metric.getActiveFraction(),
                 "work has started, but not time has elapsed yet, so the busy fraction should be 0");
 
         clock.tick(Duration.ofSeconds(1)); // time == 3
         assertEquals(
                 0.5,
-                metric.getBusyFraction(),
+                metric.getActiveFraction(),
                 "1 second of work, 1 second of idle, so the busy fraction should be 0.5");
-        metric.finishedWork();
-        assertEquals(0.5, metric.getBusyFraction(), "no time has elapsed, so the value should still be 0.5");
+        metric.deactivate();
+        assertEquals(0.5, metric.getActiveFraction(), "no time has elapsed, so the value should still be 0.5");
 
         clock.tick(Duration.ofSeconds(1)); // time == 4
         assertEquals(
                 0.33,
-                metric.getBusyFraction(),
+                metric.getActiveFraction(),
                 0.01,
                 "1 second of work, 2 seconds of idle, so the busy fraction should be 0.33");
 
         clock.tick(Duration.ofSeconds(1)); // time == 5
         assertEquals(
                 0.25,
-                metric.getBusyFraction(),
+                metric.getActiveFraction(),
                 0.01,
                 "1 second of work, 3 seconds of idle, so the busy fraction should be 0.25");
 
         clock.tick(Duration.ofSeconds(1)); // time == 6
         assertEquals(
                 0.20,
-                metric.getBusyFraction(),
+                metric.getActiveFraction(),
                 0.01,
                 "1 second of work, 4 seconds of idle, so the busy fraction should be 0.20");
-        metric.startingWork();
-        assertEquals(0.20, metric.getBusyFraction(), "no time has elapsed, so the value should still be 0.2");
+        metric.activate();
+        assertEquals(0.20, metric.getActiveFraction(), "no time has elapsed, so the value should still be 0.2");
 
         clock.tick(Duration.ofSeconds(1)); // time == 7
         assertEquals(
                 0.33,
-                metric.getBusyFraction(),
+                metric.getActiveFraction(),
                 0.01,
                 "2 second of work, 4 seconds of idle, so the busy fraction should be 0.33");
 
         clock.tick(Duration.ofSeconds(1)); // time == 8
         assertEquals(
                 0.43,
-                metric.getBusyFraction(),
+                metric.getActiveFraction(),
                 0.01,
                 "3 second of work, 4 seconds of idle, so the busy fraction should be 0.43");
         assertEquals(
                 0.43,
                 metric.getAndReset(),
                 0.01,
                 "the snapshot should contain the same value returned by getBusyFraction()");
```

Comment:
assertion messages throughout this class mention old method names

Reply:
fixed